xds/server.go

```diff
@@ -52,9 +50,7 @@ var (
 	newGRPCServer = func(opts ...grpc.ServerOption) grpcServer {
 		return grpc.NewServer(opts...)
 	}
-
-	drainServerTransports = internal.DrainServerTransports.(func(*grpc.Server, string))
-	logger                = grpclog.Component("xds")
+	logger = grpclog.Component("xds")
```

Comment:
Optional: Given that you are touching this line, it might make more sense to move this to `xds/xds.go`.